melodic/distribution.yaml

```diff
@@ -3766,6 +3766,31 @@ repositories:
       url: https://github.com/tu-darmstadt-ros-pkg/hector_slam.git
       version: melodic-devel
     status: maintained
+  heifu:
+    doc:
+      type: git
+      url: https://heifu:G2SzWMcydduaaXwttrxA@gitlab.pdmfc.com/drones/ros1/heifu.git
+      version: debian
+    release:
+      packages:
+      - heifu_bringup
+      - heifu_description
+      - heifu_diagnostic
+      - heifu_interface
+      - heifu_mavros
+      - heifu_msgs
+      - heifu_safety
+      - heifu_scripts_firmware
+      - heifu_simple_waypoint
+      - heifu_tools
+      - heifu_usb
+      - scripts
+      url: https://heifu:G2SzWMcydduaaXwttrxA@gitlab.pdmfc.com/drones/ros1/heifu.git
```

Comment:
This url should be a release repository not the package repository. To make your first release please see this tutorial: http://wiki.ros.org/bloom/Tutorials/FirstTimeRelease